src/clients/authorization/policy/PolicyDetails.tsx

```diff
@@ -57,7 +57,8 @@ const COMPONENTS: {
   js: JavaScript,
 } as const;
 
-const isValidComponentType = (value: string): boolean => value in COMPONENTS;
+export const isValidComponentType = (value: string): boolean =>
```

Comment:
```suggestion
export const isValidComponentType = (value: string): boolean =>I believe the return type here can be inferred.
```suggestion
export const isValidComponentType = (value: string) =>
```
```